message Publication {
 required string car_model = 1;
 required string production_date = 2;
 required int32 horsepower = 3;
 required string color = 4;
 required int32 max_speed = 5;
 optional int32 ts = 6;
}